lte/gateway/python/magma/mobilityd/serialize_utils.py

```diff
@@ -47,8 +47,26 @@ def _desc_state_proto_to_str(proto):
         IPDesc.REAPED: ip_descriptor.IPState.REAPED,
         IPDesc.RESERVED: ip_descriptor.IPState.RESERVED,
     }
-    state = state_map[proto]
-    return state
+    return state_map[proto]
+
+
+def _desc_type_str_to_proto(type):
+    proto_map = {
```

Comment:
Since this functions will be called to convert each IP desc in the mobility store objects, can we define the proto / type maps outside the function since anyway these will not change?

Reply:
done.